### PR TITLE
Tutorial notifications

### PR DESCRIPTION
**Pull request by PrimozGodec**
At the Friday meeting long ago we have been talking about having a notification with the link to a tutorial for new users. Here is my idea of the text. Any comments welcome. 

orange-notification-feed looks great. I was just a little confused with `local_config` options. Is there a possibility to make documentation with all options available?

### CODE REVIEW

File: notifications/16102019-tutorialOffer.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
```
Comment (PrimozGodec) [inline notifications/16102019-tutorialOffer.yml:3]: @irgolic is that OK? So the notification will appear in the first 5 startsups of Orange?

File: notifications/16102019-tutorialOffer.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+icon: "canvas/icons/announcement.png"
+title: "Orange tutorials"
+text: "Do you want to learn how to use Orange? Check our video tutorials."
+link: "https://www.youtube.com/watch?v=HXjnDIgGDuI&list=PLmNPvQr9Tf-ZSDLwOzxpvY-HrE0yv-8Fy"
+accept_button_label: 'Tutorials'
```
Comment (irgolic) [inline notifications/16102019-tutorialOffer.yml]: Perhaps this button should say `YouTube` to signify an external link?
Either that or just a simple `Ok`.

File: notifications/16102019-tutorialOffer.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
+icon: "canvas/icons/announcement.png"
+title: "Orange tutorials"
+text: "Do you want to learn how to use Orange? Check our video tutorials."
```
Comment (irgolic) [inline notifications/16102019-tutorialOffer.yml]: Consider changing this to:

`New to Orange? Get started by checking out our video tutorials!`

File: notifications/16102019-tutorialOffer.yml (hunk 1 above)
Comment (irgolic) [reply notifications/16102019-tutorialOffer.yml:3]: Yes, until it is responded to (either accept or reject is pressed, as opposed to dismiss).

File: notifications/16102019-tutorialOffer.yml (hunk 2 above)
Comment (PrimozGodec) [reply notifications/16102019-tutorialOffer.yml]: Ok goes well with the text you suggested

File: notifications/16102019-tutorialOffer.yml (hunk 3 above)
Comment (PrimozGodec) [reply notifications/16102019-tutorialOffer.yml]: That is better.

File: notifications/16102019-tutorialOffer.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
+icon: "canvas/icons/announcement.png"
```
Comment (irgolic) [inline notifications/16102019-tutorialOffer.yml]: This icon is currently still a placeholder actually... Change it to `canvas/icons/statistics-request.png`, which is the Orange icon.

Comment (PrimozGodec) [reply notifications/16102019-tutorialOffer.yml]: Ok, it is changed now

**Pull request merged**